### PR TITLE
Support setting a default value for `OpaqueType`

### PR DESCRIPTION
**Pull request by davidpdrsn**
See https://github.com/EmbarkStudios/ark-modules/issues/8273 (issue for closed source repo) for the use case.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,15 +16,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - `StructVariant`
     - `TupleStructVariant`
 - **added:** Add `EnumType::variants_len` ([#94])
+- **added:** Support setting a default value for `OpaqueType`
 
 [#92]: https://github.com/EmbarkStudios/mirror-mirror/pull/92
-[#93]: https://github.com/EmbarkStudios/mirror-mirror/pull/93
 [#94]: https://github.com/EmbarkStudios/mirror-mirror/pull/94
 
 # 0.1.4 (08. February, 2023)
```
Comment (bnjbvr) [inline CHANGELOG.md:24]: preexisting: is the release date correct here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,15 +16,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - `StructVariant`
     - `TupleStructVariant`
 - **added:** Add `EnumType::variants_len` ([#94])
+- **added:** Support setting a default value for `OpaqueType`
```
Comment (bnjbvr) [inline CHANGELOG.md:19]:
```suggestion
- **added:** Support setting a default value for `OpaqueType` ([#97])
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,15 +16,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
 [#92]: https://github.com/EmbarkStudios/mirror-mirror/pull/92
-[#93]: https://github.com/EmbarkStudios/mirror-mirror/pull/93
 [#94]: https://github.com/EmbarkStudios/mirror-mirror/pull/94
```
Comment (bnjbvr) [inline CHANGELOG.md:22]:
```suggestion
[#94]: https://github.com/EmbarkStudios/mirror-mirror/pull/94
[#97]: https://github.com/EmbarkStudios/mirror-mirror/pull/97
```

**Review by bnjbvr — APPROVED** [review]
Makes sense in the abstract. I've asked a question in the internal issue and this might question our internal requirement for this, but overall I think this is a good thing to have.

File: CHANGELOG.md (hunk 1 above)
Comment (davidpdrsn) [reply CHANGELOG.md]: It was not. Good catch!

**Pull request merged**